src/runner/runner.py

```
@@ -102,20 +116,45 @@ def run_idf(
+                # TimeoutExpired (caught below) if it overruns.
+                if timeout is not None:
+                    process.wait(timeout=timeout)
+                else:
+                    process.wait()
+
+                # Process has exited — drain all buffered stdout (the pipe
+                # yields until EOF, reached now that the child is gone).
+                if process.stdout is not None:
+                    for line in process.stdout:
+                        line = line.rstrip()
+                        self.logger.info("[EnergyPlus] {}", line)
```

Ruff (0.16.1)

[error] Too many arguments for `logging` format string

(PLE1205)